dhis-2/dhis-test-integration/src/test/java/org/hisp/dhis/tracker/Assertions.java

```diff
@@ -260,12 +255,29 @@ public static void assertNoErrorsAndNoWarnings( ImportReport report )
                 "Expected import without warnings, instead got:\n" + report.getValidationReport().getWarnings() ) );
     }
 
+    public static void assertNoErrors( ImportReport report )
+    {
+        assertNotNull( report );
+        assertEquals( Status.OK, report.getStatus(),
+            errorMessage( "Expected import with status OK, instead got:\n", report.getValidationReport() ) );
+    }
+
     public static void assertNoErrors( ValidationReport report )
     {
         assertNotNull( report );
         assertFalse( report.hasErrors(), errorMessage( "Expected no validation errors, instead got:\n", report ) );
     }
 
+    public static void assertNoErrors( ObjectBundleValidationReport report )
```

Comment:
Just thinking if this is the right place for this.
This is an assertion for metadata validation object, probably shouldn't be in tracker assertion.

Reply:
@jbee any opinion on a better place for this assertion?

Maybe in `org.hisp.dhis.dxf2.metadata.objectbundle.feedback.Assertions` in the dhis-test-integration module 🤔

Reply:
Given the dependencies I guess it is hard to move it module wise. Maybe we can use a more general package? `org.hisp.dhis.feedback` ?